src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int side;
+    private int base;
+
+    public RightTriangle(int side, int base, String color) {
+        this.side = side;
+        this.base = base;
+        this.setColor(color);
+    }
+
+    public int getSide() {
+        return side;
+    }
+
+    public int getBase() {
+        return base;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Drawing a righttrianangle");
+    }
+
+    @Override
+    public double getArea() {
+        return (side * base) / 2;
+    }
+
+    @Override
+    public String getFigureData() {
+        return "Figure: triangle, area: " + getArea()
+                + " sq. units, side: " + getSide()
+                + " sq. units, base: " + getBase()
+                + " units, color: " + getColor();
```

Comment:
The units for 'side' and 'base' are inconsistent in the 'getFigureData()' method. The 'side' mentions 'sq. units' which is incorrect as it's not an area, it should just be 'units'.